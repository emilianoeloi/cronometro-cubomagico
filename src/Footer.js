import React, { Component } from 'react';

class Footer extends Component {

  render() {
    return (
      <footer className="pure-g">
        <div className="pure-u-1 pure-u-md-1-2 pure-u-lg-1-2">
          <div className="r-box">
            <h4>
              <a target="_blank" href="https://www.youtube.com/watch?v=IrY5JjSFR8I&t=5s">Aprenda a montar: </a>
            </h4>
            <a target="_blank" href="https://www.youtube.com/user/rafaelcinoto">
              Canal do Youtube: Cinoto
            </a> | <a target="_blank" href="http://www.ws.binghamton.edu/fridrich/cube.html">
              Fridrich
            </a>
          </div>
        </div>
        <div className="pure-u-1 pure-u-md-1-2 pure-u-lg-1-2">
          <div className="r-box">
            <h4>
              <a target="_blank" href="https://pt.wikipedia.org/wiki/C%C3%B3digo_aberto">
                Contribua:
              </a>
            </h4>
            <a target="_blank" href="https://github.com/emilianoeloi/cronometro-cubomagico">Projeto no Github</a>
          </div>
        </div>
        <div className="pure-u-1">
          <div className="r-box">
            <p>
              <a href="https://github.com/emilianoeloi/cronometro-cubomagico/blob/master/LICENSE">
                APACHE-2.0
              </a> 2016-2017 Croncube www.croncube.com.br 1.1.6
            </p>
          </div>
        </div>
      </footer>
    )
  }
}

export default Footer;
