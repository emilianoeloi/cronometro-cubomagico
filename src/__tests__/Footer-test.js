import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../Footer';

test('test Footer', () => {
  const component = renderer.create(
    <Footer />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
