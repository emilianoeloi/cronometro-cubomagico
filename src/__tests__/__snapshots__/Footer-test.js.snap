exports[`test test Footer 1`] = `
<footer
  className="pure-g">
  <div
    className="pure-u-1 pure-u-md-1-2 pure-u-lg-1-2">
    <div
      className="r-box">
      <h4>
        <a
          href="https://www.youtube.com/watch?v=IrY5JjSFR8I&t=5s"
          target="_blank">
          Aprenda a montar: 
        </a>
      </h4>
      <a
        href="https://www.youtube.com/user/rafaelcinoto"
        target="_blank">
        Canal do Youtube: Cinoto
      </a>
       | 
      <a
        href="http://www.ws.binghamton.edu/fridrich/cube.html"
        target="_blank">
        Fridrich
      </a>
    </div>
  </div>
  <div
    className="pure-u-1 pure-u-md-1-2 pure-u-lg-1-2">
    <div
      className="r-box">
      <h4>
        <a
          href="https://pt.wikipedia.org/wiki/C%C3%B3digo_aberto"
          target="_blank">
          Contribua:
        </a>
      </h4>
      <a
        href="https://github.com/emilianoeloi/cronometro-cubomagico"
        target="_blank">
        Projeto no Github
      </a>
    </div>
  </div>
  <div
    className="pure-u-1">
    <div
      className="r-box">
      <p>
        <a
          href="https://github.com/emilianoeloi/cronometro-cubomagico/blob/master/LICENSE">
          APACHE-2.0
        </a>
         2016-2017 Croncube www.croncube.com.br 1.1.6
      </p>
    </div>
  </div>
</footer>
`;
